tests/Unit/Utilities/Test_Functional.cpp

```diff
@@ -22,56 +159,131 @@ void test_get_argument() {
   CHECK(GetArgument<3, 1>{}(-2, 8, -10) == 8);
   CHECK(GetArgument<3, 2>{}(-2, 8, -10) == -10);
 }
-void test_identity() {
-  CHECK(Identity{}(-2) == -2);
-  CHECK(Identity{}(-2.0) == -2.0);
-}
-void test_minus() {
-  CHECK(Minus<>{}(1, -2) == 3);
-  CHECK(Minus<>{}(1, -2.0) == 3.0);
-}
-void test_multiplies() {
-  CHECK(Multiplies<>{}(1, -2) == -2);
-  CHECK(Multiplies<>{}(1, -2.0) == -2.0);
-}
-void test_negate() {
-  CHECK(Negate<>{}(10) == -10);
-  CHECK(Negate<>{}(-10.0) == 10.0);
-}
-void test_plus() {
-  CHECK(Plus<>{}(1, -2) == -1);
-  CHECK(Plus<>{}(1, -2.0) == -1.0);
-}
-void test_sqrt() {
-  CHECK(Sqrt<>{}(3.0) == sqrt(3.0));
-}
-void test_square() {
-  CHECK(Square<>{}(3.0) == 9.0);
-  CHECK(Square<>{}(-3.0) == 9.0);
-}
 
-void test_composition() {
```

Comment:
So the goal of `test_composition` was to make sure the basic design worked in terms of arity tracking, expecially for more complex examples like on 57, 58. It would be nice to retain that at least so that if `Functional` or something gets changed, we still know arity tracking is working.